app/views/queue_items/index.html.haml

```diff
@@ -4,30 +4,32 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |item|
+        = bootstrap_form_tag update_queue_path
```

Comment:
You are missing a `do` here, which is why you are getting that error. Haml errors can be cryptic unfortunately. I also had to add `url: ` before the path to get it to work. I then faced an issue with your `hidden_field`, but I didn't continue debugging it.